bin/upgrade/DEV/upgrade.mongo.50.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
 
 #check if authentication is required
-isAuth=$(mongo --eval "db.getUsers()" | grep -v "connecting" | grep "auth")
```

Comment:
set `isAuth` to `0` here and remove `else` part of next `if` block

Reply:
updated the code block 👍